packages/fxa-settings/src/pages/ResetPasswordWithRecoveryKeyVerified/index.test.tsx

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import ResetPasswordWithRecoveryKeyVerified from '.';
+import { logViewEvent, usePageViewEvent } from '../../lib/metrics';
+
+jest.mock('../../lib/metrics', () => ({
+  logViewEvent: jest.fn(),
+  usePageViewEvent: jest.fn(),
+}));
+
+describe('ResetPasswordWithRecoveryKeyVerified', () => {
+  it('renders default content as expected', () => {
+    render(<ResetPasswordWithRecoveryKeyVerified />);
+
+    const newAccountRecoveryKeyButton = screen.getByText(
```

Comment:
I don't think we need to set these to `const` and then check that it's in the doc, `getByText` should magically fail by itself 😎:

```
screen.getByText('whatever');
```

Reply:
I think it's good to check for it in the doc so that it's clear what the expected behavior is by reading the test? Good idea w the SVG!

Reply:
Hmm maybe it's more clear if someone doesn't know that `getBy` tests automatically, but we also have test duplication here essentially, and more lines to maintain. 🤔

Reply:
I would still lean pretty heavily towards keeping it in, since the `getBy` error message only expresses what the failure without the test intention. I'm happy to add a comment in to make that more explicit!

Reply:
Nah that's fine, we do it both ways in the codebase. I guess we could ask the team which way is preferred for consistency in the future but it's not a blocker by any means and it's fine to leave as-is.

Reply:
that's a great idea!